react-charts/bar-chart/step5.md

```diff
@@ -1,34 +1,68 @@
-Now we'll add tooltips to the bar chart...
+Add axis labels to the bar chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
-
-2) Locate the code that looks like the following:
+2) <strong>Locate the code for the `<Chart>` component.</strong>
 
 <pre class="file">
 &lt;Chart
+  containerComponent={&lt;ChartVoronoiContainer
+  labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`} /&gt;}
   domainPadding={{ x: [30, 25] }}
   height={250}
-  width={600}
-&gt;
+  width={600}&gt;
 </pre>
 
-3) Add the following property to that section:
+3) <strong>Add the `padding` property to that section:</strong>
+
+In this case, give it the following values. More padding will be added to the
+bottom and left properties to accommodate the axis labels.

```

Comment:
Replace with: In this case, give it the following values. More padding will be added to the bottom and left properties to accommodate the axis labels.